models/public/mozilla-deepspeech-0.6.1/mozilla-deepspeech-0.6.1.md

```diff
@@ -34,74 +34,74 @@ Increasing beam_width improves WER metric and slows down decoding.  Speech recog
 
 ### Original Model
 
- * **Audio MFCC coefficients**, name: `input_node` , shape: [1x16x19x26], format: [BxNxTxC], where:
+ 1. **Audio MFCC coefficients**, name: `input_node`, shape: `1, 16, 19, 26`, format: `B, N, T, C`, where:
```

Comment:
```suggestion
 1.  Audio MFCC coefficients, name: `input_node`, shape: `1, 16, 19, 26`, format: `B, N, T, C`, where:
```
For aligning with other mozilla-deepspeech models

Reply:
Done